cosmos/stride.json

```diff
@@ -49,6 +49,10 @@
       "coinDenom": "stSTARS",
       "coinMinimalDenom": "stustars",
       "coinDecimals": 6
+    },
+      "coinDenom": "stLUNA",
```

Comment:
@shellvish Could you add `{` between 52 and 53 lines?

Reply:
Wow, so sorry, not sure how I missed that. Thanks a lot for the flag!